### ISSUE TITLE
Uncaught AssertionError: html-pdf: Failed to load PhantomJS module. You have to set the path to t...

### ISSUE THREAD

**Issue by gdgjson** [travs/markdown-pdf#177]
[Enter steps to reproduce:]

1. ...
2. ...

**Atom**: 1.21.2 x64
**Electron**: 1.6.15
**OS**: Mac OS X 10.12.6
**Thrown From**: [markdown-pdf](https://github.com/travs/markdown-pdf) package 2.1.0

### Stack Trace

Uncaught AssertionError: html-pdf: Failed to load PhantomJS module. You have to set the path to the PhantomJS binary using 'options.phantomPath'

```
At assert.js:85

AssertionError: html-pdf: Failed to load PhantomJS module. You have to set the path to the PhantomJS binary using 'options.phantomPath'
    at /packages/markdown-pdf/node_modules/mdpdf/node_modules/html-pdf/lib/pdf.js:38:3)
    at /packages/markdown-pdf/node_modules/mdpdf/node_modules/html-pdf/lib/index.js:10:14)
    at /packages/markdown-themeable-pdf/lib/markdown-themeable-pdf.js:483:27
    at tryToString (fs.js:426:3)
    at FSReqWrap.readFileAfterClose [as oncomplete] (fs.js:413:12)
```

### Commands

```
     -0:09.8.0 markdown-themeable-pdf:export (input.hidden-input)
```

### Non-Core Packages

```
markdown-img-paste 0.3.9 
markdown-pdf 2.1.0 
markdown-preview-plus 2.4.16 
markdown-themeable-pdf 1.2.0 
pdf-view 0.59.0 
```

**Comment by dynamicheart** [comment]
brew install PhantomJS

**Comment by travs** [comment]
Unfortunately yes, it seems you will have to install phantomJS manually sometimes on a mac system. 
This occurs in legacy/fallback mode, and the regular mode will be upgraded to a new version of `mdpdf` which uses puppeteer instead of a browser renderer, so this shouldn't happen once that is complete.

For now, perhaps try installing phantomJS from your package manager, or via `npm install -g phantomjs`, or try re-installing the package without any firewall/proxy/vpn.

Duplicate of https://github.com/travs/markdown-pdf/issues/114 https://github.com/travs/markdown-pdf/issues/105 https://github.com/travs/markdown-pdf/issues/144 https://github.com/travs/markdown-pdf/issues/107
